workflows/rollback-test.json

```diff
@@ -5,7 +5,7 @@
         "type": "download-file",
         "inputs": [
             "http://nginx/sample.tif",
-            "/home/data"
+            "/opt/geoserver_data/sample.tif"
```

Comment:
you can remove this fix, i just opened a PR fixing all workflows here: https://github.com/klips-project/klips-sdi/pull/47